.github/workflows/pr_build_block_bench.yml

```diff
@@ -0,0 +1,29 @@
+name: '"Build block" benchmark'
```

Comment:
look into the names of other workflows. They use `perf` nomenclature in naming

Reply:
Also, it seems this could be added as another job inside `Benchmark Block execution` instead of new workflow?

Reply:
Done! About merging the workflows: I think it's best to do it this way since I'm using another github action to post the results and the diff.